src/lib.rs

```diff
@@ -1420,14 +1428,14 @@ mod test {
     use bdk::miniscript::bitcoin::network::constants::Network::Testnet;
     #[cfg(all(feature = "reserves", feature = "electrum"))]
     use bdk::{
-        blockchain::{noop_progress, ElectrumBlockchain},
-        database::MemoryDatabase,
-        electrum_client::Client,
-        Wallet,
+        blockchain::ElectrumBlockchain, database::MemoryDatabase, electrum_client::Client, Wallet,
     };
     use std::str::{self, FromStr};
     use structopt::StructOpt;
 
+    #[cfg(all(feature = "reserves", feature = "electrum",))]
+    use crate::bdk::SyncOptions;
```

Comment:
Nit. This can be added into an import statement few lines above which already imports stuffs from `bdk` if `reserve` and `electrum` is enabled.